Rebuild-Black-Business/RBB-Website
# Feature/update link styles across the site

# Describe your PR
This updates the link styles across the site 


Fixes #314 


## Pages/Interfaces that will change
- /allies
-/businesses
-footer 
-/about


### Screenshots / video of changes

<img width="279" alt="Screen Shot 2020-08-25 at 00 43 18" src="https://user-images.githubusercontent.com/53913806/91072474-2b90fc80-e66c-11ea-9687-d7c146d93d32.png">
<img width="596" alt="Screen Shot 2020-08-25 at 00 43 59" src="https://user-images.githubusercontent.com/53913806/91072523-36e42800-e66c-11ea-8518-577adff74e80.png">


Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)
hovering on call-to-action links before the update : 
<img width="272" alt="Screen Shot 2020-08-25 at 00 48 09" src="https://user-images.githubusercontent.com/53913806/91072997-f2a55780-e66c-11ea-9f6b-3a2e3edc708c.png">
<img width="601" alt="Screen Shot 2020-08-25 at 00 49 12" src="https://user-images.githubusercontent.com/53913806/91073016-f933cf00-e66c-11ea-85cc-7cdbbeb3b1d3.png">


## Steps to test

1. Go to the /businesses or the allies page 
2. hover on any of the call-to-action links on each of the business or allies cards
3. you will observe the links changing to a purple color(which is defined by the new link styles standard)
